### PR TITLE
Update wechaty.md

### PR DESCRIPTION
**Pull request by su-chang**
make example run right

### CODE REVIEW

File: root/api/wechaty.md [hunk 1]
```diff
@@ -197,7 +195,7 @@ bot.on('room-join', (room, inviteeList, inviter) => {
 
 bot.on('room-leave', (room, leaverList) => {  
   const nameList = leaverList.map(c => c.name()).join(',')  
-  console.log(`Room ${room.topic()} lost member ${nameList}`)
+  console.log(`Room ${await room.topic()} lost member ${nameList}`)
```
Comment (lijiarui) [inline root/api/wechaty.md]: await should use in async function.

File: root/api/wechaty.md [hunk 2]
```diff
@@ -186,7 +184,7 @@ bot.on('scan', (url, code) => {  console.log(`[${code}] Scan ${url} to login.` )
 ​
 bot.on('room-join', (room, inviteeList, inviter) => {  
   const nameList = inviteeList.map(c => c.name()).join(',')  
-  console.log(`Room ${room.topic()} got new member ${nameList}, invited by ${inviter}`)
+  console.log(`Room ${await room.topic()} got new member ${nameList}, invited by ${inviter}`)
```
Comment (lijiarui) [inline root/api/wechaty.md]: await should use in async function.

File: root/api/wechaty.md [hunk 3]
```diff
@@ -195,9 +193,9 @@ bot.on('room-join', (room, inviteeList, inviter) => {
 ```typescript
 // room-leave Event will emit when someone leave the room.​
 
-bot.on('room-leave', (room, leaverList) => {  
+bot.on('room-leave', async (room, leaverList) => {
```
Comment (windmemory) [inline root/api/wechaty.md]: There is a `remover`, which is the third argument in the callback function. Let's put it in the doc.

Comment (su-chang) [reply root/api/wechaty.md]: The `remover` is an optional param, so i think its no need to add in an example first, however, after ur mentioned, I find there was no doc for the params of callback function in `bot.on(Event, Listener)`, same question in Room Listener.

Shall we add some detail like this to describe the Listener:
```
public on (event: 'room-leave',   listener: string | ((this: Wechaty, room: Room, leaverList: Contact[], remover?: Contact) => void))             : this
```

**Pull request merged**